utils/modular_model_converter.py

```diff
@@ -828,17 +871,19 @@ def leave_SimpleStatementLine(self, original_node, updated_node):
         parent_node = self.get_metadata(cst.metadata.ParentNodeProvider, original_node)
         if m.matches(parent_node, m.Module()):
             if m.matches(updated_node, m.SimpleStatementLine(body=[m.Import()])):
-                if updated_node not in self.all_imports:
-                    self.all_imports.append(updated_node)
+                for k in updated_node.body[0].names:
+                    import_name = self.python_module.code_for_node(k.name)
+                    self.all_imports[import_name] = updated_node
```

Comment:
No need to iterate on `k` and use a dict: simpler to add to the whole node in a set as before imo (`code_for_node` is called in `leave_Module` anyway)